public/src/main/java/com/revenuecat/purchases/models/StoreProduct.kt

```diff
@@ -32,6 +32,11 @@ interface StoreProduct : Parcelable {
 
     /**
      * Title of the product.
+     *
+     * If you are using Google subscriptions with multiple base plans, this title
+     * will be the same for every subscription duration (monthly, yearly, etc) as
+     * base plans don't have their own titles. Google suggests using the duration
+     * as a way to title base plans.
```

Comment:
I wonder should we add a property named `titleWithDuration` or something in case devs want to use that?

Reply:
This hard due to localization 😔 I think this is _still_ an option but needs some more planning on best way to get it done